packages/plugins/plugin-adjust/src/AdjustPlugin.tsx

```diff
@@ -16,9 +16,11 @@ export class AdjustPlugin extends DestinationPlugin {
   type = PluginType.destination;
   key = 'Adjust';
 
+  private adjust: AdjustConfig | undefined = undefined;
   private settings: SegmentAdjustSettings | null = null;
 
   update(settings: SegmentAPISettings, _: UpdateType) {
+    const analytics = this.analytics ?? undefined;
```

Comment:
I think the null-coalescing isn't necessary? if `this.analytics` is `undefined` it will remain `undefined`